core/tests/protractor_utils/ExplorationPlayerPage.js

```diff
@@ -125,6 +127,10 @@ var ExplorationPlayerPage = function() {
     waitFor.pageToFullyLoad();
   };
 
+  this.closeSuggestionModal = function() {
```

Comment:
@rakshitkumarcse, @DubeySandeep: should this be renamed to `clickOnCloseSuggestionModalButton` since this is what the function does? Also, why don't we have `waitFor.elementToBeClickable(..)` and `waitFor.pageToFullyLoad()` like other clickOn.. functions? Any specific reasoning?

Reply:
done :)